src/Synapse/Validator/Constraints/I18n/BelongsToEntity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Synapse\Validator\Constraints\I18n;
+
+use Synapse\Validator\Constraints\BelongsToEntity as ParentConstraint;
+
+class BelongsToEntity extends ParentConstraint
+{
+    public $message = 'DOES_NOT_BELONG_TO_ENTITY';
```

Comment:
All of these messages should actually be constants for easy reference in testing and elsewhere.
```
const DOES_NOT_BELONG_TO_ENTITY = 'DOES_NOT_BELONG_TO_ENTITY';
public $message = self::DOES_NOT_BELONG_TO_ENTITY;
```